tests/helpers/test-methods.js

```diff
@@ -37,37 +37,42 @@ module.exports = {
       }
     }
   },
-  checkHomepageCanonicalUrls: async ([response, frontendURL]) => {
+  /**
+   * Checks the canonical URL within the given CE-API response.
+   *
+   * @param {Response} response The given CE-API response
+   * @param canonicalURL The expected URL
+   * @returns {Promise<void>}
+   */
+  expectCanonicalUrl: async (response, canonicalURL) => {
```

Comment:
I had issues in the past when having the function arguments passed like this. Can't remember why, but it only worked if it was passed as an array. See other funcs with multiple arguments. 

Maybe this works now, has it been tested like this?

Reply:
yes, I was wondering about that weird array syntax. Yes, I tested it, works fine, so let's better start with regular params?

Reply:
@petarbasic yes, I tested